bedrock/mozorg/templates/mozorg/contribute/index.html

```diff
@@ -150,6 +150,23 @@ <h2 class="section-title">{{ _('Not ready to dive in just yet?') }}</h2>
 
 {% endblock contrib_content %}
 
+{% block cta %}
+
+  {% set localized_surveys = {'en-US': 'GHJYW6D'} %}
+
+  {% if LANG in localized_surveys %}
+    <div id="survey-message">
+      <span class="survey-invite">
+        {{ _('Hello! Would you be willing to take a minute to answer a few questions?') }}
```

Comment:
Note that this string isn't localized, so I'd suggest that we hardcode it as opposed to wrap it in the L10n function call here. When it comes to adding localized surveys, we'll need to provide a translations here. But we can worry about that when the time comes.